apple/swift-docc
# Remove error for mixed symbol graph formats.

Bug/issue #, if applicable: rdar://114405721 rdar://115438349 #702

## Summary

This error has proven to be too strict. Due to synthesized symbols, exported symbols, and other cases it's possible that one of the extensions symbol graph files won't contain any `extension` symbols resulting in a failure even though all symbol graphs were generated in the same build workflow with the same flags (possibly even from the same call to `swift-frontend`).

## Dependencies

None

## Testing

- Build documentation for [FluentKit](https://github.com/vapor/fluent-kit) as described in #702.

- There shouldn't be an error about mismatched symbol graph file formats.  

- Inspect the build log and look for `swift-frontend` calls with the `-emit-symbol-graph` flag. 

- Verify that all those calls are also passed `-emit-extension-block-symbols`. This shows that there wasn't a format mismatch.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
